package.json

```diff
@@ -19,5 +19,7 @@
   },
   "peerDependencies": {
     "@nebula.js/stardust": ">=1.2.3"
-  }
+  },
+  "packageManager": "pnpm@^6",
```

Comment:
I know this has already been merged. But his is not a valid format for `packageManager`. You have to use explicit version, ex: `pnpm@6.0.0`

https://github.com/nodejs/corepack?tab=readme-ov-file#when-authoring-packages

Reply:
Correct. Renovate fixed it for us #74